app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -102,11 +84,13 @@ protected void onStart() {
             }
         });
 
-        btnLogin.setOnClickListener(v -> {
+        binding.loginContent.btnLogin.setOnClickListener(view -> {
+            if (!validator.validate())
```

Comment:
this return true for `https://....com/api/v1/` which we are using currently. Although we can just use `https://....com` as a base URL because HostSelectionInterceptor takes care of the suffix section using original url passed to retrofit from `Constants.BASE_URL`.